Automattic/woocommerce-payments
# Added tests around saving dispute for editing

Fixes https://github.com/Automattic/woocommerce-payments/issues/1487

#### Changes proposed in this Pull Request

This PR adds a new test around saving a Dispute for editing. Additionally, this adds a new dispute credit card and some helper functions around dispute flows.

#### Testing instructions

* Make sure the CI tests pass
* Run the new test locally and verify it passes in either headless or non-headless mode:
    * `npm run test:e2e-dev ./tests/e2e/specs/disputes/save-for-editing.spec.js`
    * `npm run test:e2e ./tests/e2e/specs/disputes/save-for-editing.spec.js`
